### PR TITLE
Lesson44 spring boot mvc thymeleaf

### PR DESCRIPTION
**Pull request by nait13**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1,9 @@
+spring.mvc.view.prefix=/WEB-INF/views/
```
Comment (DHrytsuk) [inline src/main/resources/application.properties]: Is it correct? And do you need it in general?

File: src/main/resources/templates/students/edit.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+</head>
+<body>
+<section th:class="section">
+    <div class="container">
+        <div class="row">
+            <div class="container container--xs">
+                <div class="woocommerce">
+                    <div id="signup_div_wrapper" class="bloc-center">
+                        <h3 class="mb-1 text-center">Update student from TMS C27</h3>
+                        <form th:method="PATCH" th:action="@{/student/{id}(id = ${student.getId()})}" th:object="${student}" class="register row g-3 shadow-sm">
```
Comment (DHrytsuk) [inline src/main/resources/templates/students/edit.html]: no PATCH method for mvc

File: src/main/resources/templates/students/index.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+                <th>ID</th>
+                <th>First Name</th>
+                <th>Last Name</th>
+                <th>Actions</th>
+            </tr>
+            <tr th:if="${students.isEmpty()}">
+                <td class="text-center" colspan="7">No Records found. Add some...!</td>
+            </tr>
+            <tr th:each="student : ${students}">
+                <td class="align-middle" th:text="${student.getName()}"></td>
```
Comment (DHrytsuk) [inline src/main/resources/templates/students/index.html]:
```suggestion
                <td class="align-middle" th:text="${student.name}"></td>
```

File: src/main/java/org/lesson43/dao/StudentDAO.java [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+public class StudentDAO {
+    private PostgresDriverManager postgresDriverManager;
+
+    @Autowired
+    public void setPostgresDriverManager(PostgresDriverManager postgresDriverManager) {
```
Comment (DHrytsuk) [inline src/main/java/org/lesson43/dao/StudentDAO.java]: Setters should be placed at the end of the class

File: src/main/java/org/lesson43/dao/StudentDAO.java [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+                System.out.println(student);
+            }
+            return studentList;
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return null;
+    }
+
+    public Student show(int id) {
```
Comment (DHrytsuk) [inline src/main/java/org/lesson43/dao/StudentDAO.java]:
```suggestion
    public Student getStudent(int id) {
```

File: src/main/java/org/lesson43/controller/StudentsController.java [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+import java.util.List;
+
+@Controller
+@RequestMapping("/student")
+public class StudentsController {
+    private StudentDAO studentDAO;
+
+    @Autowired
+    public void setStudentDAO(StudentDAO studentDAO) {
```
Comment (DHrytsuk) [inline src/main/java/org/lesson43/controller/StudentsController.java]: Setter at the end of the class

File: src/main/java/org/lesson43/dao/StudentDAO.java (hunk 4 above)
Comment (nait13) [reply src/main/java/org/lesson43/dao/StudentDAO.java]: ok

**Review by DHrytsuk — APPROVED** [review]

**Pull request closed without merging**